### PR TITLE
code coverage 98%

### PR DESCRIPTION
**Pull request by harshithaputtaswamy**

### CODE REVIEW

File: tests/test_fyle/test_helpers.py [hunk 1]
```diff
@@ -32,7 +46,21 @@ def test_get_request(mocker):
...
+        'apps.fyle.helpers.requests.get',
+        return_value=Response(
+            {
+                'message': 'Get request'
+            },
+            status=status.HTTP_400_BAD_REQUEST
+        )
+    )
+    try:
+        get_request(url='sdfghjk', params={'sample': True}, refresh_token='srtyu')
```
Comment (NileshPant1999) [inline tests/test_fyle/test_helpers.py:63]: where are asserts 😕

File: tests/test_workspaces/test_tasks.py [hunk 2]
```diff
@@ -77,17 +80,21 @@ def test_email_notification(db):
...
         workspace_id=workspace_id 
     ).first()
+    run_email_notification(workspace_id=workspace_id)
+
+    attribute = ExpenseAttribute.objects.filter(workspace_id=workspace_id, value='user5@fyleforgotham.in').first()
+    attribute.delete()
 
     ws_schedule.enabled = True
-    ws_schedule.emails_selected = ['ashwin.t@fyle.in']
+    ws_schedule.additional_email_options = [{'email': 'user5@fyleforgotham.in', 'name': 'Ashwin'}]
```
Comment (NileshPant1999) [inline tests/test_workspaces/test_tasks.py:97]: again asserts 😕

File: tests/test_fyle/test_helpers.py (hunk 1 above)
Comment (harshithaputtaswamy) [reply tests/test_fyle/test_helpers.py:63]: Here we are mocking the get call response so why do we have to add asserts here?

Comment (harshithaputtaswamy) [reply tests/test_fyle/test_helpers.py:55]: 400 response just raises exception, status code 200 we are mocking the get call response, have added assert for those

File: tests/test_workspaces/test_tasks.py (hunk 2 above)
Comment (harshithaputtaswamy) [reply tests/test_workspaces/test_tasks.py:103]: added

**Review by NileshPant1999 — APPROVED** [review]

File: tests/test_fyle/test_helpers.py [hunk 3]
```diff
@@ -6,13 +6,23 @@
 
 
 def test_post_request(mocker):
+    mocker.patch(
+        'apps.fyle.helpers.requests.post',
+        return_value=mock.MagicMock(status_code=200, text="{'key': 'dfghjk'}")
```
Comment (NileshPant1999) [inline tests/test_fyle/test_helpers.py:11]: single quotes

File: tests/test_fyle/test_helpers.py [hunk 4]
```diff
@@ -22,32 +32,38 @@ def test_post_request(mocker):
 
 
 def test_get_request(mocker):
+    mocker.patch(
+        'apps.fyle.helpers.requests.get',
+        return_value=mock.MagicMock(status_code=200, text="{'key': 'dfghjk'}")
```
Comment (NileshPant1999) [inline tests/test_fyle/test_helpers.py:37]: single quotes

**Pull request merged**